coderd/database/dbfake/dbfake.go

```diff
@@ -76,29 +76,29 @@ type BuildBuilder struct {
 	resources []*sdkproto.Resource
 }
 
-func WorkspaceBuildBuilder(t testing.TB, db database.Store, ws database.Workspace) BuildBuilder {
-	return BuildBuilder{t: t, db: db, ws: ws}
+func NewWorkspaceBuildBuilder(t testing.TB, db database.Store, ws database.Workspace) WorkspaceBuildBuilder {
+	return WorkspaceBuildBuilder{t: t, db: db, ws: ws}
 }
 
-func (b BuildBuilder) Pubsub(ps pubsub.Pubsub) BuildBuilder {
+func (b WorkspaceBuildBuilder) Pubsub(ps pubsub.Pubsub) WorkspaceBuildBuilder {
 	//nolint: revive // returns modified struct
 	b.ps = ps
 	return b
 }
 
-func (b BuildBuilder) Seed(seed database.WorkspaceBuild) BuildBuilder {
+func (b WorkspaceBuildBuilder) Seed(seed database.WorkspaceBuild) WorkspaceBuildBuilder {
 	//nolint: revive // returns modified struct
 	b.seed = seed
 	return b
 }
 
-func (b BuildBuilder) Resource(resource *sdkproto.Resource) BuildBuilder {
+func (b WorkspaceBuildBuilder) Resource(resource ...*sdkproto.Resource) WorkspaceBuildBuilder {
 	//nolint: revive // returns modified struct
-	b.resources = append(b.resources, resource)
+	b.resources = append(b.resources, resource...)
 	return b
 }
 
-func (b BuildBuilder) Do() database.WorkspaceBuild {
+func (b WorkspaceBuildBuilder) Do() database.WorkspaceBuild {
```

Comment:
out of curiosity: wasn't that always `Build()`?

Reply:
It's usually `Build()` but I'm attempting to avoid a clash with WorkspaceBuild.  I don't mind too much either way.